src/operator-docs/creation/fromEvent.ts

```diff
@@ -1,6 +1,251 @@
+    },
+    {
+      name: 'options',
+      type: 'EventListenerOptions',
+      attribute: 'optional',
+      /* tslint:disable:max-line-length */
+      description: `Options to pass through to the <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener">addEventListener</a> function.
+       This is either an options object {capture?: boolean, passive?: boolean, once?: boolean } or a boolean to represent capturing.`
+      /* tslint:enable:max-line-length */
+    },
+    {
+      name: 'selector',
+      type: 'SelectorMethodSignature',
+      attribute: 'optional',
+      description: `An optional function to map the arguments from the event handler to a single value.`
+    }
+  ],
+  marbleUrl: 'http://reactivex.io/rxjs/img/fromEvent.png',
+  shortDescription: {
+    description: `
+    Creates an Observable that emits events of a specific type coming from the given event target.
+    `
+  },
+  walkthrough: {
+    description: `
+    <p>
+    Creates a new Observable based on a given target or a collection of targets, and a specific event type.
+    <span class="markdown-code">fromEvent</span> allows an options object or a selector method to be passed for extra customization.
+    </p>
+    <p>
+    The event <span class="markdown-code">target</span> is an object with methods for registering event handler functions.
+    Valid objects are either a DOM EventTarget, a Node.js EventEmitter,
+     a JQuery-style event target, a DOM NodeList, or a DOM HTMLCollection.
+    These types are described in detail below.
+    </p>
+    <p>
+    Note that the event target is checked through duck typing. In other words
+    no matter what kind of object you have and no matter what environment you work in,
+    you can safely use fromEvent on that object if it exposes any of the described methods (provided
+    of course they behave as described).
+    For example if a Node.js library exposes an event target with the same method names as a DOM EventTarget, fromEvent is still
+    a good choice.
+    </p>
+    <p>
+    If your event target does not match any of the objects listed,
+     you should use <a href="/#/operators/fromEventPattern" class="markdown-code">fromEventPattern</a>, which can be used on arbitrary APIs.
+    </p>
+    <p>
+    Every time a subscription is made to the resulting Observable,
+    the <span class="markdown-code">next</span> function of the Subscriber object is registered as the event handler
+    to the event target on the given event type. When that event fires, the value
+    passed as the first argument to the registered function will be emitted by that same Observable.
+    When the Observable gets unsubscribed, the event handler will be unregistered from the event target.
+    </p>
+
+  <p>
+  Note that if the event target would normally (without RxJS) call the registered event handler with more than one argument,
```

Comment:
maybe add an exampe event here

Reply:
I've added an example just below this paragraph. I think it makes more sense to put it there (so it doesn't stop the flow of the text with an example)